README.md

```diff
@@ -16,14 +16,24 @@ In order to successfully start payload jobs:
 5. Optional: add to the START expression with `GLIDEIN_Start_Extra`. This is useful to limit
    the pilot to only run certain jobs.
 
+In addition, you will be able to run more OSG jobs if you provide CVMFS.  You can do this
+in two ways:
+
+1. Mount CVMFS on the host and bind-mount it into the container by adding `-v /cvmfs:/cvmfs:shared`.
+   This is the preferred mechanism.
+2. Use cvmfsexec as described in [the cvmfsexec section below](#cvmfs-without-a-bind-mount-using-cvmfsexec).
+
+Note: Supporting Singularity jobs inside the container will require the capabilities
```

Comment:
No, this is still confused.  The same options needed for cvmfsexec work for singularity.  You either need to list the capabilities, use --privileged, or use the options for unprivileged.  I'm sorry I didn't make that clear; in fact I didn't realize that the discussion was distinct.

I suggest listing the 3 options here and not mention them under cvmfsexec.

Reply:
OK, I ended up skipping --privileged and just using the capabilities for both the bind-mount- and the cvmfsexec-based commands, and said that for cvmfsexec you need to add `--device=/dev/fuse`. I rebuilt the container to get cvmfsexec 4.8 and it works fine.

Do the changes look good?